go-controller/pkg/cluster/master.go

```diff
@@ -74,6 +86,55 @@ func (cluster *OvnClusterController) StartClusterMaster(masterNodeName string) e
 	return nil
 }
 
+func (cluster *OvnClusterController) ReInitClusterNode() error {
+	// setup node
+	var err error
+	var subnet *netIPNet
+
+	nodes, err := cluster.Kube.GetNodes()
+	if err != nil {
+		logrus.Errorf("Error in getting k8s nodes: %v", err)
+		return err
+	}
+	for _, node := range nodes.Items {
+		sub, ok := node.Annotations[OvnHostSubnet]
+		_, subnet, err = net.ParseCIDR(sub)
+		if err != nil {
+			logrus.Errorf("Invalid hostsubnet found for node %s - %v",
+				node.Name, err)
+			return err
+		}
+
+		err = setupOVNNode(node.Name, config.Kubernetes.APIServer,
```

Comment:
I think you can skip calling setupOVNNode() as it is local node dependent. For e.g., the token you get here may not be one the node will receive. Since the node has already set this up, you don't need to reset it. Anyways, the function sets these things locally - so it does not make much sense to do it anyway.